packages/block-editor/tsconfig.json

```diff
@@ -25,6 +25,7 @@
 		{ "path": "../is-shallow-equal" },
 		{ "path": "../keycodes" },
 		{ "path": "../notices" },
+		{ "path": "../preferences" },
```

Comment:
Did tooling catch this or did you add it manually?

I'm wondering if we need to review other packages. It looks like 8 packages depend on preferences, but maybe these are the only two packages that have a tsconfig.

```
$ grep '@wordpress/preferences' packages/*/package.json -l
packages/preferences-persistence/package.json
packages/interface/package.json
packages/preferences/package.json
packages/editor/package.json
packages/edit-site/package.json
packages/edit-widgets/package.json
packages/edit-post/package.json
packages/block-editor/package.json
packages/customize-widgets/package.json
```

Reply:
I added it manually after looking for packages with tsconfig.json depending on it.